.github/workflows/tests.yml

```diff
@@ -83,7 +83,18 @@ jobs:
 
       - run: /bin/bash uninstall.sh -f >/dev/null
 
-      - run: /bin/bash -c "$(cat install.sh)"
```

Comment:
I think we still need a test that successfully runs the install script without arguments.

Reply:
There already is a test at line 78 for no argmuents. And the new test only replaces the protocol of git remotes.